test/build-tasks-sample/build-tasks-sample.sln

```diff
@@ -0,0 +1,28 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "contract", "src\contract.csproj", "{524B2C6C-4354-42E7-A911-F55D877D7F0E}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "contract-test", "test\contract-test.csproj", "{1E87FB71-E3AD-444C-884D-E511758F9E9F}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{524B2C6C-4354-42E7-A911-F55D877D7F0E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{524B2C6C-4354-42E7-A911-F55D877D7F0E}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{524B2C6C-4354-42E7-A911-F55D877D7F0E}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{524B2C6C-4354-42E7-A911-F55D877D7F0E}.Release|Any CPU.Build.0 = Release|Any CPU
+		{1E87FB71-E3AD-444C-884D-E511758F9E9F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{1E87FB71-E3AD-444C-884D-E511758F9E9F}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{1E87FB71-E3AD-444C-884D-E511758F9E9F}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{1E87FB71-E3AD-444C-884D-E511758F9E9F}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+EndGlobal
```

Comment:
Remove this and add to `neo-express.sln` in a new folder under `test` called `samples`?

Reply:
I find this directory are more like an example to show how to use the neo-test tools, even not a part of unit tests.
 Maybe we should move it the root directory with a `samples` folder? It does not make sence to be placed here.

Reply:
Sounds good, that makes more sense.